doc/source/rllib/rl-modules.rst

```diff
@@ -177,6 +208,17 @@ Other default model settings
 For LSTM-based configurations and specific settings for continuous action output layers,
 see :py:class:`~ray.rllib.core.rl_module.default_model_config.DefaultModelConfig`.
 
+.. note::
+
+    To auto-wrap your default model with an extra LSTM layer to allow your model to learn in
```

Comment:
Does this really wrap my model or is it simply using a different encoder?

Reply:
Good point! It wraps the encoder, only. Will fix.